permissions/plugin-mina-sshd-api.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: "mina-sshd-api"
+github: &GH "jenkinsci/mina-sshd-api-plugin"
+paths:
+- "org/jenkins-ci/plugins/mina-sshd-api/mina-sshd-api-parent"
```

Comment:
you'll need to fiddle with the paths in a follow-up

Reply:
@Dohbedoh why the old-style `groupId` https://github.com/Dohbedoh/mina-sshd-api-plugin/blob/a37c0e5c79cc51f6bb4029088cc022f791289d9f/pom.xml#L12? For new plugins we tend to use `io.jenkins.plugins`.

Reply:
@jglick I didn't know. Is it still possible to change this ? if yes how ? First I open a PR here on `repository-permissions-updater` and then I make the changes on the repo ?

Reply:
If you have not released the plugin yet, it is still possible. You can change the plugin repo and RPU in either order so long as you do both before the first release.

Reply:
* https://github.com/jenkinsci/mina-sshd-api-plugin/pull/1
* https://github.com/jenkins-infra/repository-permissions-updater/pull/2584